docker-compose.yml

```diff
@@ -32,12 +32,10 @@ couchbase:
         - 11211
         - 18091
         - 18092
+    env_file: .env
     environment:
         - CONSUL_HOST=http://consul:8500
         - COUCHBASE_SERVICE_NAME=couchbase
-        - COUCHBASE_USER=Administrator
```

Comment:
What I've forgotten here is that we're sourcing in the `.env` file in the `start.sh` script. Which means we can use that.